### PR TITLE
Chore cleanup

### PR DESCRIPTION
**Pull request by flbulgarelli**

### CODE REVIEW

**Comment by flbulgarelli** [comment]
Why is wercker not running?

File: lib/classroom/database.rb [hunk 1]
```diff
@@ -19,10 +19,6 @@ def disconnect!
       client.try(:close)
     end
 
-    def within_each(&block)
```
Comment (aguspina) [inline lib/classroom/database.rb]: This method is useful for migrations affecting all databases

File: lib/atheneum.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+class Mumukit::Bridge
```
Comment (aguspina) [inline lib/atheneum.rb]: This is no longer necessary, should be removed from atheneum too

File: lib/tasks/resubmissions.rb [hunk 3]
```diff
@@ -1,19 +1,16 @@
...
         social_id = body['social_id']
 
-        logger.info "Processing resubmission #{social_id}"
-
+        Mumukit::Nuntius::Logger.info "Processing resubmission #{social_id}"
         Classroom::FailedSubmission.reprocess! social_id, destination
-      rescue => e
-        logger.error "Resubmission couldn't be processed #{e}. it was: #{body}"
       ensure
         Classroom::Database.disconnect!
```
Comment (aguspina) [inline lib/tasks/resubmissions.rb]: We have to remove this line

**Pull request closed without merging**